backend/app/usecase/authorizer/authorizer_test.go

```diff
@@ -0,0 +1,62 @@
+package authorizer
+
+import (
+	"testing"
+
+	"github.com/short-d/app/fw/assert"
+	"github.com/short-d/short/backend/app/entity"
+	"github.com/short-d/short/backend/app/usecase/authorizer/role"
+	"github.com/short-d/short/backend/app/usecase/repository"
+)
+
+func TestAuthorizer_hasPermission(t *testing.T) {
```

Comment:
```suggestion
func TestAuthorizer_CanCreateChange(t *testing.T) {
```

Reply:
`hasPermission` is private API. we should confuse on testing public APIs.

Reply:
I recommend we create a seperate struct called rbac (Role based access control) and make `hasPermission` it's public API so that we test it.